### PR TITLE
Emit request event when HTTP request from connectionstream

### PR DESCRIPTION
**Pull request by brechtvdv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ You can add streams by setting an eventlistener on the MergeStream instance.
 To remove a stream, just end the stream itself.
 ```javascript
 connectionsReadStream.on("data", function (connection) {
-	connectionsReadStream.addConnectionsStream(['newStream'], newConnectionsReadStream]);
+	connectionsReadStream.addConnectionsStream(['newStream', newConnectionsReadStream]);
```
Comment (pietercolpaert) [inline README.md:52]: I think we should rework this in a different pull request in which we have this instead:

``` javascript
connectionsReadStream.addConnectionsStream('newStream', newConnectionsReadStream);
```

Makes more sense code-wise

File: lib/MergeStream.js [hunk 2]
```diff
@@ -47,6 +47,10 @@ MergeStream.prototype.addConnectionsStream = function (connectionsStream) {
     }
   });
 
+  this._connectionsStreams[index].on('request', function() {
+    self.emit('request');
+  });
+
```
Comment (pietercolpaert) [inline lib/MergeStream.js:53]: I'm not going to merge this: csa.js should be oblivious about what comes in; Why do you need request to be emitted here? For all csa.js cares, there is just a stream that comes in.

If you need to count it in your demo, as I understand why you want to have an event emitted, just keep count it using the connection stream itself, not the result stream.

**Comment by pietercolpaert** [comment]
Not going to merge. See comments ↑

File: README.md (hunk 1 above)
Comment (brechtvdv) [inline README.md:52]: Will do!

**Pull request closed without merging**